controllers/managedocs_controller_test.go

```diff
@@ -805,6 +811,20 @@ var _ = Describe("ManagedOCS controller", func() {
 				}, timeout, interval).Should(BeTrue())
 			})
 		})
+		When("the ocsInitialization resource is created", func() {
+			It("should patch the ocsInitialization to enable ceph toolbox", func() {
+				ocsInit := ocsInitializationTemplate.DeepCopy()
+				Expect(k8sClient.Create(ctx, ocsInit)).Should(Succeed())
+
+				ocsInitKey := utils.GetResourceKey(ocsInit)
+
+				// Wait for the spec changes to be reverted
+				Eventually(func() bool {
+					Expect(k8sClient.Get(ctx, ocsInitKey, ocsInit)).Should(Succeed())
+					return ocsInit.Spec.EnableCephTools
+				}, timeout, interval).Should(Equal(true))
+			})
+		})
```

Comment:
Missing a test case to validate that if the field is changed we revert the change (should be very similar to the create test and can even assume that ocsInitialization was already created and the field was set by the prev test